pushd /home/emacs/build/kernel/linuxgit/makepkg/linux-git/src/linux-git; git diff drivers/input >/tmp/lei8042.patch ; popd
cp /tmp/lei8042.patch ./2500_lei8042.patch


cat <<EOF
Remember to remove this part(manually!) from the patch because another patch is applying it and it will thus conflict/fail:

@@ -220,6 +220,11 @@ static int i8042_wait_write(void)
 	return -(i == I8042_CTL_TIMEOUT);
 }
 
+#define udbg(format, arg...)							\
+	do {									\
+			printk(KERN_DEBUG KBUILD_MODNAME ": " format,	\
+			       ##arg);	\
+	} while (0)
 /*
  * i8042_flush() flushes all data that may be in the keyboard and mouse buffers
  * of the i8042 down the toilet.
@@ -235,10 +240,10 @@ static int i8042_flush(void)
 	spin_lock_irqsave(&i8042_lock, flags);
 
 	while ((str = i8042_read_status()) & I8042_STR_OBF) {
-		if (count++ < I8042_BUFFER_SIZE) {
+		if (count++ < 66/*I8042_BUFFER_SIZE*/) { // https://bugzilla.kernel.org/show_bug.cgi?id=92881
 			udelay(50);
 			data = i8042_read_data();
-			dbg("%02x <- i8042 (flush, %s)\n",
+			udbg("%02x <- i8042 (flush, %s)\n",
 			    data, str & I8042_STR_AUXDATA ? "aux" : "kbd");
 		} else {
 			retval = -EIO;
EOF
